Sky Editor/Application.xaml.vb

```diff
@@ -7,6 +9,16 @@ Class Application
 
     Private Async Sub Application_Startup(sender As Object, e As StartupEventArgs) Handles Me.Startup
         StartupHelpers.EnableErrorDialog()
+
+        'Checks if Java is installed'
+        If File.Exists("C:\Program Files (x86)\Common Files\Oracle\Java\javapath\java.exe") Then
```

Comment:
Java might not be installed in this specific path. A better way would probably be to just handle the relevant exception when we try to start Java and .Net can't find it.

Also, the change should be made in the ROM editor project, since it's currently the only thing using Java: https://github.com/evandixon/SkyEditor.ROMEditor

Reply:
Ok thanks for the help!